apps/health_campaign_field_worker_app/lib/widgets/progress_bar/beneficiary_progress.dart

```diff
@@ -59,7 +60,8 @@ class _BeneficiaryProgressBarState extends State<BeneficiaryProgressBar> {
             .where((element) =>
                 element.dateOfRegistrationTime.isAfter(gte) &&
                 (element.isDeleted == false || element.isDeleted == null) &&
-                element.dateOfRegistrationTime.isBefore(lte))
+                element.dateOfRegistrationTime.isBefore(lte) &&
+                element.auditDetails!.createdBy == context.loggedInUserUuid)
```

Comment:
why do we need to filter it multiple times, it is already fetching based on user id during search